Runtime/Editor/LootLockerExtensionResponses.cs

```diff
@@ -0,0 +1,122 @@
+using System.Collections.Generic;
```

Comment:
General for this file is that I think it's a bit messy in what is defined here. I would sort the request definitions into one section, response definitions into one, and so on. Make sure that all actual responses inherit LootLockerResponse so that we can error check.